### PR TITLE
[PAN-OS] Edit documentation 

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6724

## Description
Edit documentation for ***pan-os-list-rules*** command.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GuyAfik** [comment]
@rshunim please add it to the `query` argument description in the `pan-os-list-rules` command if possible

**Comment by rshunim** [comment]
> @rshunim please add it to the `query` argument description in the `pan-os-list-rules` command if possible

I added in the 'query' argument and also in the command's description

**Comment by GuyAfik** [comment]
> > @rshunim please add it to the `query` argument description in the `pan-os-list-rules` command if possible
> 
> I added in the 'query' argument and also in the command's description

i meant in the yml too

File: Packs/PAN-OS/ReleaseNotes/1_17_2.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Documentation and metadata improvements.
```
[the remark this answers is not in the document]
Comment (dorschw) [reply Packs/PAN-OS/ReleaseNotes/1_17_2.md:6]: I agree

File: Packs/PAN-OS/Integrations/Panorama/README.md [hunk 2]
```diff
@@ -3049,23 +3049,24 @@ Deprecated. Retrieves traffic log query data by job id.
 ### pan-os-list-rules
 ***
 Returns a list of predefined Security Rules.
+(Note: When passing a query, all other arguments are overridden. Make sure the query includes all the filters you want)
```
Comment (dorschw) [inline Packs/PAN-OS/Integrations/Panorama/README.md:3052]:
```suggestion

**Note**: When passing a query, all other arguments are overridden. Make sure the query includes all necessary filters.
```

**Pull request merged**